posix.mak

```diff
@@ -142,24 +142,24 @@ ALL_FILES = $(ALL_FILES_BUT_SITEMAP) $(DOC_OUTPUT_DIR)/sitemap.html
 
 # Pattern rulez
 
-$(DOC_OUTPUT_DIR)/%.html : %.dd $(DDOC)
+$(DOC_OUTPUT_DIR)/%.html : %.dd $(DDOC) $(DMD)
```

Comment:
Added a bunch of $(DMD) and $(DMD_REL) dependencies, such that those are built automatically if needed.